common/predictive-text/worker/worker-interfaces.ts

```diff
@@ -132,6 +132,41 @@ interface WorkerInternalModelConstructor {
   new(...modelParameters: any[]): WorkerInternalModel;
 }
 
-interface WorkerInternalWordBreaker {
-  break(text: string): string[]; // 
-}
\ No newline at end of file
+/**
+ * A simple word breaking function takes a phrase, and splits it into "words",
+ * for whatever definition of "word" is usable for the language model.
+ *
+ * For example:
+ *
+ *   getText(breakWordsEnglish("Hello, world!")) == ["Hello", "world"]
+ *   getText(breakWordsCree("ᑕᐻ ᒥᔪ ᑮᓯᑲᐤ ᐊᓄᐦᐨ᙮")) == ["ᑕᐻ", "ᒥᔪ ᑮᓯᑲᐤ""", "ᐊᓄᐦᐨ"]
+ *   getText(breakWordsJapanese("英語を話せますか？")) == ["英語", "を", "話せます", "か"]
```

Comment:
I assume these are all hypothetical examples at present, right?  I don't see anything directly supporting use of actual Cree or Japanese for wordbreaking.

Reply:
Correct. These are hypothetical examples to communicate the semantics of this interface. Plus, I think `breakWordsEnglish()` and `breakWordsCree()` can both be handled by the Unicode default word boundary specification.